apps/www/src/registry/default/plate-ui/table-cell-element.tsx

```diff
@@ -40,6 +44,7 @@ const TableCellElement = React.forwardRef<
       className={cn(
         'relative overflow-visible border-none bg-background p-0',
         hideBorder && 'before:border-none',
+        bgStyleValue ? 'bg-[--cellBackground]' : 'bg-background',
```

Comment:
Can you share a screenshot of the CSS this produces in the web inspector for a custom `cellBackground`? If it uses `background` instead of `background-color`, this is vulnerable to attacks as discussed in the previous PR.

Reply:
![C74F7651-E908-4F81-9C04-588340DAB622_4_5005_c](https://github.com/udecode/plate/assets/39378793/b9584c71-71e6-4187-9f7c-b8b40dc28149)

Reply:
Great! It should be fine, then.